packages/types/src/hooks.ts

```diff
@@ -0,0 +1 @@
+export type Hooks = 'startTransaction' | 'transactionFinish';
```

Comment:
m: Should be `Hook` I think, not plural? As otherwise it is weird e.g.:

```
const hook: Hooks = 'startTransaction';
```

Reply:
Or maybe even `HookType` or `HookName` or something like this, as `Hook` seems a bit as if it would be a function type or so?